common/djangoapps/util/tests/test_db.py

```diff
@@ -121,6 +122,7 @@ class MigrationTests(TestCase):
     """
 
     @override_settings(MIGRATION_MODULES={})
+    @skip("OldMongo Deprecation")
```

Comment:
It generated 1 migration in lms `channel_integrations/sap_success_factors/migrations/0005_rename_sap_success_enterpr_8151d6_idx_sap_success_enterpr_cbd802_idx.py`